site/en/tutorials/text/word2vec.ipynb

```diff
@@ -612,7 +612,10 @@
         "id": "_KlwdiAa9crJ"
       },
       "source": [
-        "![word2vec_negative_sampling](https://tensorflow.org/tutorials/text/images/word2vec_negative_sampling.png)"
+        "![word2vec_negative_sampling](https://lh3.googleusercontent.com/FK0tsefdo39ffDhUKctpdeRz8E_ul5CD9A7VWFbzT9bSeRt2fy3tZ9lr88ZFQHTJEZpgAL8ELiET5Gv90fEiVTlYuOcOk46SpY7Efj-UF4Xwb7bVhe8ujx3gTAnzqXbMXBQRbFmkJA=w2400)"
```

Comment:
We can't include this external link.

Can you download the file, and add it to this pr, replacing the old file at: 

en/tutorials/text/images/word2vec_negative_sampling.png

Reply:
I was not sure if I could change the old file given the structure of the tensorflow website so I added a new file with the changed image and created a PR. https://github.com/tensorflow/docs/pull/2000